bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -148,6 +148,8 @@ spec:
             - name: ZOO_PROMETHEUS_METRICS_PORT_NUMBER
               value: {{ .Values.metrics.port | quote }}
             {{- end }}
+            - name: POD_NAME
+              subPathExpr: $(POD_NAME)
```

Comment:
Shouldn't it be

```
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.name
```

Reply:
You are absolutely right @javsalgar . I copy pasted wrong section from another chart. Thank you.

Reply:
done.